pytest_doctestplus/plugin.py

```diff
@@ -115,6 +115,12 @@ def pytest_addoption(parser):
                   type='linelist',
                   default=[])
 
+    parser.addini("doctest_subpackage_requires",
+                  "A list of paths to skip if requirements are not satisfied. Each item in the list "
+                  "should have the syntax path=req1;req2.",
```

Comment:
A little worried that people might copy-paste that period.
```suggestion
                  "should have the syntax path=req1;req2",
```